components/BookCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface BookCardProps {}
```

Comment:
Let's remove this and add it once we need it. No premature code please.

Also, let's use `type`s over `interface`s.

Interfaces have stung me in the past in multiple ways. For example: 

- You can declare the same `interface` multiple times and TS will just go with it and create some weird intersection. All `type` names are expected to be unique like variables.
- Interfaces are more common with OOP (`class Thing implements MyInterface`). I'd like this project to be more _functional_ than object-oriented.
- Interfaces and types are composable the same way, but types have unique constraints.
- VS Code only: when hovering on a type, you get a FULL DESCRIPTION of its schema. When hovering on an interface, you just see `interface MyInterface` without any idea what it actually is.

Reply:
We should add this to a wiki page, just in case :)